modules/nextflow/src/main/groovy/nextflow/container/ContainerConfig.groovy

```diff
@@ -87,8 +87,10 @@ class ContainerConfig extends LinkedHashMap {
         final eng = getEngine()
         if( !eng )
             return null
-        if( eng=='docker' || eng=='podman' )
-            return '--rm --privileged'
+        if( eng=='docker' )
+            return '--rm --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined'
```

Comment:
no need to add SYS_ADMIN capability with docker. Only `--rm --device /dev/fuse --security-opt apparmor=unconfined --security-opt seccomp=unconfined` should work. Notice missing seccomp unconfined.